javascript/javascript2/week2/doublingOfNumbers.js

```diff
@@ -0,0 +1,11 @@
+
+function getDoubleOddNumber () {
+  let result = doubledOddNumber;
+  return result
```

Comment:
You are missing a `;` here. Also, you could consider just returning `doubledOddNumber` without assigning to a variable first.